app/gui2/src/components/visualizations/ScatterplotVisualization.vue

```diff
@@ -859,39 +867,44 @@ config.setToolbar([
 </script>
 
 <template>
-  <div ref="containerNode" class="ScatterplotVisualization">
-    <svg :width="width" :height="height">
-      <g ref="legendNode"></g>
-      <g :transform="`translate(${margin.left}, ${margin.top})`">
-        <defs>
-          <clipPath id="clip">
-            <rect :width="boxWidth" :height="boxHeight"></rect>
-          </clipPath>
-        </defs>
-        <g ref="xAxisNode" class="axis-x" :transform="`translate(0, ${boxHeight})`"></g>
-        <g ref="yAxisNode" class="axis-y"></g>
-        <text
-          v-if="data.axis.x.label"
-          class="label label-x"
-          text-anchor="end"
-          :x="xLabelLeft"
-          :y="xLabelTop"
-          v-text="data.axis.x.label"
-        ></text>
-        <text
-          v-if="showYLabelText"
-          class="label label-y"
-          text-anchor="end"
-          :x="yLabelLeft"
-          :y="yLabelTop"
-          v-text="data.axis.y.label"
-        ></text>
-        <g ref="pointsNode" clip-path="url(#clip)"></g>
-        <g ref="zoomNode" class="zoom" :width="boxWidth" :height="boxHeight" fill="none">
-          <g ref="brushNode" class="brush"></g>
+  <div v-if="errorMessage && errorMessage.length > 0">
```

Comment:
This conditional can be inserted without adding a layer of `div`s to the DOM, either by:
- Remove the outer divs; put the `v-if` and `v-else` on the inner divs.
- Or: Use `<template v-if` and `<template v-else`.